com.unity.formats.fbx/Documentation~/exporting.md

```diff
@@ -189,6 +189,7 @@ In addition, the FBX Exporter exports the following animated attributes:
   - World Up Vector (Aim Constraint)
   - Up Vector (Aim Constraint)
   - Aim Vector (Aim Constraint)
+- Blendshapes
```

Comment:
This is quite uninformative, let me know if you want me to add a section with more information. Or perhaps the DMM team may want to include something specific.

Reply:
feel free to add a section for it if there is more information to add

Reply:
I think it's fine as-is. At most I'd add "(since version 4.1.0)" so people know it's a recent addition.